contrib/containers/deploy/Dockerfile.GitHubActions.Release

```diff
@@ -23,6 +23,8 @@ RUN apt-get update && \
     ca-certificates \
     && rm -rf /var/lib/apt/lists/*
 
+RUN echo https://github.com/${GITHUB_REPOSITORY}/releases/download/v${TAG}/dashcore-${TAG}-$arch.tar.gz
```

Comment:
nit: I'd like to see "..." for each string that has any \$ 

becase if $TAG is "aba ; rm -rf /* ; " than it would be run `echo` first and `rm` after and some garbage at the end of line

just for sanity and safety

Reply:
```suggestion
RUN echo "https://github.com/${GITHUB_REPOSITORY}/releases/download/v${TAG}/dashcore-${TAG}-$arch.tar.gz"
```